roles/setup/tasks/setup-container.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: Enable linger for {{ receptor_user }}
```

Comment:
We could check lingering status for the sake of proper idempotency/changed status:

```
- name: Gather linger status
  ansible.builtin.shell: "set -o pipefail && /usr/bin/loginctl show-user {{ receptor_user }} | sed -n -e 's/^Linger=//p'"
  changed_when: false
  register: linger
```

Then this block would have `when: linger.stdout != "yes"` and `changed_when: true`

Reply:
When I try to use this block to check linger status before it is configured, I get the following error - 
```
[Failed to get user: User ID 1001 is not logged in or lingering](fatal: [remote-execution]: FAILED! => {"changed": false, "cmd": "set -o pipefail && /usr/bin/loginctl show-user awx | sed -n -e 's/^Linger=//p'", "delta": "0:00:00.011165", "end": "2023-11-29 15:33:42.765649", "msg": "non-zero return code", "rc": 1, "start": "2023-11-29 15:33:42.754484", "stderr": "Failed to get user: User ID 1001 is not logged in or lingering", "stderr_lines": ["Failed to get user: User ID 1001 is not logged in or lingering"], "stdout": "", "stdout_lines": []})
```
Unable to get this working yet, but will try and come back to this note